fiftyone/server/files/operations.py

```
@@ -0,0 +1,153 @@
+
+from .errors import FileOperationError, FileOperationErrorCode
+from .paths import (
+    check_and_resolve_path,
+    check_feature_enabled,
+    get_unique_path,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _sync_resolve_and_open(normalized: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_sync_resolve_and_open`

(ANN202)